hack/ocm-minikube.sh

```diff
@@ -160,6 +163,8 @@ spoke_add()
 	# Error from server (InternalError): Internal error occurred: failed calling webhook "managedclustermutators.admission.cluster.open-cluster-management.io": the server is currently unable to handle the request
 	set -e
 	date
+	kubectl --context ${hub_cluster_name} patch managedclusters/${1} -p $(jq -cn --arg clname "$1" '{"metadata":{"labels":{"name":$clname}}}') --type=merge
```

Comment:
This is also in the ramen script: https://github.com/RamenDR/ramen/blob/2b8b96c5b673e13cbabacfd559cb62728ca8e2ae/hack/ocm-minikube-ramen.sh#L98
Based on its inclusion here I presume it is an OCM requirement, more so than a ramen one.  And if so, I wonder why OCM does not do it.  @BenamarMk any thoughts?

Reply:
OCM upstream does not do it yet, although this is how its placement works.

I did not notice it in the ramen deploy part (had not got that far). It is better in the ocm setup as this is an expectation from the OCM setup phase.

Reply:
issue tracker to address managedcluster labeling: https://github.com/open-cluster-management-io/multicloud-operators-subscription/issues/16